sdk/communication/communication-phone-numbers/samples-dev/siprouting/manageSipRoutingConfiguration.ts

```diff
@@ -64,9 +73,22 @@ export async function main() {
     console.log(`Route ${route.name} with pattern ${route.numberPattern}`);
     console.log(`Route's trunks: ${route.trunks?.join()}`);
   }
+
+  // Clear configuration
+  await client.setRoutes([]);
+  await client.setTrunks([]);
 }
 
 main().catch((error) => {
   console.log("The sample encountered an error:", error);
   process.exit(1);
 });
+
+function getUniqueFqdn() {
+  const length = 12;
+  let random = 0;
+  do {
+    random = Math.floor(Math.random() * 10 ** length);
+  } while (random < 10 ** (length - 1));
+  return `sample.${random}.com`;
```

Comment:
As an alternative, how about using the `uuid` package and generating something like `${v4()}.example.com`?

Reply:
Used UUIDs instead.

import { v4 as uuid } from "uuid";
const firstTrunkFqdn = `sample.${uuid()}.com`;